src/components/MAPI/organizations/AccessControl/guides/BindRolesToSubjectsGuide.tsx

```diff
@@ -0,0 +1,120 @@
+              external: true,
+            },
+          ]}
+        />
+      }
+      {...props}
+    >
+      <Box direction='column' gap='small'>
+        <LoginGuideStep />
+        <CLIGuideStep
+          title='2. Bind a Role to a user or group to grant access to resources of this organization'
+          command={`kubectl create rolebinding example \\
+          -n ${namespace} --user example@acme.org --role read-all`}
+        >
+          <Text>
+            <strong>Note:</strong> Replace <code>example</code> with a unique
+            name for the <code>RoleBinding</code>. Replace{' '}
+            <code>example@acme.org</code> with the actual user&apos;s ID and{' '}
+            <code>read-all</code> with the name of the <code>Role</code> you
+            want to bind. Be aware that the role must exist in this namespace
+          </Text>
+          <Text>
+            In order to grant access to a <strong>group</strong> of users, use
+            the <code>--group</code> flag instead of the <code>--user</code>{' '}
+            flag.
+          </Text>
+        </CLIGuideStep>
+        <CLIGuideStep
+          title='3. Bind a ClusterRole to a user or group to grant access to resources of this organization'
+          command={`kubectl create rolebinding example \\
+           -n ${namespace} --user example@acme.org --clusterrole read-all`}
+        >
+          <Text>
+            <strong>Note:</strong> Replace <code>example</code> with a unique
+            name for the <code>RoleBinding</code>. Replace{' '}
+            <code>example@acme.org</code> with the actual user&apos;s ID and{' '}
+            <code>read-all</code> with the name of the <code>ClusterRole</code>{' '}
+            you want to bind.
+          </Text>
+          <Text>
+            In order to grant access to a <strong>group</strong> of users, use
+            the <code>--group</code> flag instead of the <code>--user</code>{' '}
+            flag.
+          </Text>
+        </CLIGuideStep>
+        <CLIGuideStep
+          title='4. Bind a Role or CclusterRole to a ServiceAccount or grant access to resources of this organization'
+          command={`kubectl create rolebinding example \\
+          -n ${namespace} --serviceaccount ${namespace}:my-sa --role read-all`}
+        >
+          <Text>
+            <strong>Note:</strong> Replace <code>example</code> with a unique
+            name for the <code>RoleBinding</code>. Replace <code>my-sa</code>{' '}
+            with the name of the <code>ServiceAccount</code> and{' '}
+            <code>read-all</code> with the name of the <code>Role</code> you
+            want to bind. Be aware that the role must exist in this namespace.
+          </Text>
+          <Text>
+            In order to bind the <code>ServiceAccount</code> to a{' '}
+            <code>ClusterRole</code> instead of a <code>Role</code>, use the{' '}
+            <code>--clusterrole</code> flag instead of <code>--role</code>.
```

Comment:
```suggestion
            In order to bind the service account to a{' '}
            cluster role instead of a role, use the{' '}
            <code>--clusterrole</code> flag instead of <code>--role</code>.
```